DigitalLearningSolutions.Web/Styles/myAccount/myAccount.scss

```diff
@@ -19,12 +9,6 @@
   height: 150px;
 }
 
-.left-button-mobile-margin-bottom {

```

Comment:
This was already moved to index.scss in another ticket, but it was not removed from here, so I removed it now.

Reply:
Missed that, thanks